File: src/service/Validator.java
package service;

public class Validator {
    public boolean validate(String input) {
        if (input == null || input.isEmpty()) {
            throw new IllegalArgumentException("Invalid input");
        }
        return true;
    }

    public String sanitize(String input) {
        return input.replaceAll("[<>]", "");
    }

    public boolean checkLength(String input, int max) {
        return input.length() <= max;
    }
}

File: src/core/validator.ts

export function validate(input: string) {
  if (!input) throw new Error('Invalid');
  return true;
}

export function checkSchema(schema: any) {
  return schema && typeof schema === 'object';
}

export function sanitize(input: string) {
  return input.replace(/[<>]/g, '');
}
